source/mainnet/net/snippets/delegation-faq.rst

```diff
@@ -41,7 +50,8 @@ For delegators
 
 .. dropdown:: Is there a “cool-down” period when delegating and how does it work?
 
-    Yes, for certain changes there is a :term:`cool-down<cool-down period>` period built into the pool system. The cool-down period is three weeks. During the cool-down period it is not possible to change the delegated amount or move the delegated amount to a different staking pool. The cool-down period is activated when you decrease the delegated amount or stop delegation entirely.
+    If you decrease or remove delegations, the funds are not released to your account until after a :term:`cool-down<cool-down period>` period.
+    The cool-down period is three weeks. The cool-down period is activated when you decrease the delegated amount or stop delegation entirely.
```

Comment:
Is this in line with P7 updates?

Reply:
Documentation updates relating to P7, including this, have been reviewed by Thomas so it should be.